### PR TITLE
Try triggering another pipeline

### PR DESCRIPTION
**Pull request by jared-improbable**

### CODE REVIEW

File: scripts/GetDependentBranchNames/Program.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                    throw new Exception("The BUILDKITE_BRANCH environment variable is empty or not set.");
+                }
+
+                foreach (var dep in Dependencies)
+                {
+                    var lines = new List<string>();
+                    shell.Run("git", "ls-remote", "--heads", dep.Repo, currentBranch)
+                        .RedirectTo(lines)
+                        .RedirectStandardErrorTo(Console.Error)
+                        .Wait();
```
Comment (jamiebrynes7) [inline scripts/GetDependentBranchNames/Program.cs:47]: Will this throw an exception is the command exits with non-zero code?

File: scripts/GetDependentBranchNames/Program.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+                    var remoteBranch = lines.Any() && lines.First().Contains(currentBranch) ? currentBranch : "master";
```
Comment (jamiebrynes7) [inline scripts/GetDependentBranchNames/Program.cs:49]: `lines.Any()` could be not-needed here? 

```csharp
var remoteBranch = lines.FirstOrDefault()?.Contains(currentBranch) ? currentBranch : "master";
```

(I'm unsure if C# can coerce the `bool?` to `bool` here)

**Review by jamiebrynes7 — APPROVED** [review]
Generally LGTM

File: scripts/GetDependentBranchNames/Program.cs (hunk 1 above)
Comment (jared-improbable) [reply scripts/GetDependentBranchNames/Program.cs:47]: Yep, see the initialization of `shell` above.

File: scripts/GetDependentBranchNames/Program.cs (hunk 2 above)
Comment (jared-improbable) [reply scripts/GetDependentBranchNames/Program.cs:49]: I wrote it like that originally, but didn't like it for whatever reason

**Pull request merged**